### DISCUSSION TITLE
利用dnsopd api，自动将最快IP更新至dnspod

### DISCUSSION THREAD

**Discussion by zhzhyu-com** [XIU2/CloudflareSpeedTest#567] [分享]
本人使用的是腾讯云的dns解析，略微修改了一下源代码，可以将筛选出的最快ip通过api更新至dnspod
方法：1、在dnspod上解析一个记录，类型为A，记录值随便（如192.168.1.1）
           2、需要去腾讯云申请api密钥，并将密钥及域名信息写入config.json
           3、启动程序
注：除了生成result.csv文件这一功能被我剔除了，其余功能参数都有保留
[CloudflareSpeedTest.zip](https://github.com/user-attachments/files/18272320/CloudflareSpeedTest.zip)
[config.json](https://github.com/user-attachments/files/18272325/config.json)
初学golang，有哪些不完善的还请指正

**Comment by zhzhyu-com** [comment]
是腾讯云的api密钥，不是dnspod的，不要申请错了

**Comment by zhzhyu-com** [comment]
[CloudflareSpeedTest.zip](https://github.com/user-attachments/files/18272389/CloudflareSpeedTest.zip)
上传错了，重新上传一个

**Comment by XIU2** [comment]
这种需求，用 CloudflareST 搭配外部脚本实现最好也最方便（这种简单的需求 AI 都能写），而且已经有人做了：

https://github.com/XIU2/CloudflareSpeedTest/tree/master/script#-cfst_dnspodsh
